docs/maintainer-guide/governance.md

```diff
@@ -37,49 +37,94 @@ To become a Committer:
+
+No more than 1/3 of the TSC members may be affiliated with the same employer. If removal or resignation of a TSC member, or a change of employment by a TSC member, creates a situation where more than 1/3 of the TSC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more TSC members affiliated with the over-represented employer(s).
 
-* Contributions can be reviewed by other Reviewers, but this is not explicitly required.
-* May merge external pull requests for accepted issues upon reviewing and approving the changes.
-* May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one Committer/Reviewer comment stating they've looked at the code.)
 
+TSC members have additional responsibilities over and above those of a Committer. These responsibilities ensure the smooth running of the project. TSC members are expected to review code contributions, approve changes to this document, manage the copyrights within the project outputs, and attend regular TSC meetings.
 
-To become a Reviewer:
+TSC members fulfill all requirements of Committers, and also:
 
-* Reviewers are expected to work in a helpful and collaborative way with the community.
+* May merge external pull requests for accepted issues upon reviewing and approving the changes.
+* May merge their own pull requests once they have collected the feedback they deem necessary. (No pull request should be merged without at least one Committer/TSC member comment stating they've looked at the code.)
+
+To become a TSC member:
+
+* TSC members are expected to work in a helpful and collaborative way with the community.
 * Have given good feedback on others' submissions and displayed an overall understanding of the code quality standards for the project.
 * Have submitted a minimum of 50 qualifying pull requests.
 
-A Committer is invited to become a Reviewer by existing Reviewers. A nomination will result in discussion and then a vote by the existing Reviewers.
+A Committer is invited to become a TSC member by existing TSC members. A nomination will result in discussion and then a decision by the TSC.
+
+#### TSC Meetings
+
+The TSC meets every other week in the [TSC Meeting](https://gitter.im/eslint/tsc-meetings) chatroom. The meeting is run by
+a designated moderator approved by the TSC.
+
+Items are added to the TSC agenda which are considered contentious or
+are modifications of governance, contribution policy, TSC membership,
+or release process.
+
+The intention of the agenda is not to approve or review all patches.
+That should happen continuously on GitHub and be handled by the larger
+group of Committers.
 
-### Project Lead
+Any community member or committer can ask that something be added to
+the next meeting's agenda by logging a GitHub Issue. Anyone can add the item to the agenda by adding
+the "tsc agenda" tag to the issue.
```

Comment:
I'd prefer not to list out all the times it's appropriate to add this label, as I think there are a great deal of situations where it makes sense. Use your judgment. The general rule is that if an issue just isn't getting resolved, then flag it for the TSC to resolve.